bool led_on
bool on_food
float64 x
float64 y
float64 size
